build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 plugins {
-  id("uk.gov.justice.hmpps.gradle-spring-boot") version "4.0.1"
+  id("uk.gov.justice.hmpps.gradle-spring-boot") version "4.0.3-beta"
```

Comment:
Think this may break when the tested version has been released?

Reply:
I think the 4.0.3 will go out soon, and both versions will still be deployed and available AFAIK.